File: 1662-check-if-two-string-arrays-are-equivalent/1662-check-if-two-string-arrays-are-equivalent.java
class Solution {
    public boolean arrayStringsAreEqual(String[] word1, String[] word2) {
        String a = "";
        for(int i=0;i<word1.length;i++)
        {
            a+=word1[i];
        }
        String b="";
        for(int i=0;i<word2.length;i++)
        {
            b+=word2[i];
        }
        return a.equals(b);
    }
}

File: 1662-check-if-two-string-arrays-are-equivalent/1662-check-if-two-string-arrays-are-equivalent.cpp
class Solution {
public:
    bool arrayStringsAreEqual(vector<string>& word1, vector<string>& word2) {
        string temp1="";
        string temp2="";
        for(auto x:word1)
        {
            temp1+=x;
        }
        for(auto x:word2)
        {
            temp2+=x;
        }
        if(temp1.compare(temp2)==0)
        {
            return true;
        }
        return false;
    }
};